### PR TITLE
Final changes for release notes 6.11

### PR DESCRIPTION
**Pull request by GuiMacielPereira**
Added release notes from changes that were merged during release period.

### Description of work
Final touches for release notes for 6.11

#### Summary of work

Added missing release notes, edited index page, fixed typos and grammatical errors and added a new gif.

### To test:
Build `docs-html` and read over release notes to look for typos or bad formatting.

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

**Comment by GuiMacielPereira** [comment]
I am still waiting on Richard to make the note about the new algorithm `FindMultipleUMatrices` clearer

File: docs/source/release/v6.11.0/framework.rst [hunk 1]
```diff
@@ -88,6 +89,9 @@ New features
 - Fix python fuction ``assert_almost_equal`` to fail for non-equal workspaces.
 - The python function ``assert_almost_equal`` for testing if two modules are within a tolerance was reworked.
 
+Bugfixes
+############
+- Fixed error in the log about `load_module()` being deprecated in Python 3.12.
```
Comment (jhaigh0) [inline docs/source/release/v6.11.0/framework.rst:94]:
```suggestion
- Fixed error in the log about ``load_module()`` being deprecated in Python 3.12.
```

File: docs/source/release/v6.11.0/mantidworkbench.rst [hunk 2]
```diff
@@ -7,13 +7,13 @@ Mantid Workbench Changes
 
 New Features
 ------------
+- There is a new :ref:`Deprecation Policy <deprecation_policy>` for Mantid.
```
Comment (jhaigh0) [inline docs/source/release/v6.11.0/mantidworkbench.rst:10]: This note should be with the framework release notes

**Review by jhaigh0 — CHANGES_REQUESTED** [review]

File: docs/source/release/v6.11.0/direct_geometry.rst [hunk 3]
```diff
@@ -21,15 +21,30 @@ New features
 Bugfixes
 ############
 - :ref:`ALFView <ALFView-ref>` no longer crashs when moving the cursor over the Instrument View while loading data.
+- :ref:`ALF View <ALFView-ref>` no longer crashes when closing the interface while it is loading data.
```
[the remark this answers is not in the document]
Comment (thomashampson) [reply docs/source/release/v6.11.0/direct_geometry.rst:24]: ALFView seems more common so I'll change to that.

File: docs/source/release/v6.11.0/diffraction.rst [hunk 4]
```diff
@@ -75,15 +76,15 @@ New features
 .. figure::  ../../images/6_11_release/FindMultipleUMatrices.png
    :width: 400px
 
-- Algorithm :ref:`ShowPossibleCells <algm-ShowPossibleCells>` now has a new output property, ``Cells``, that includes the cell information in a usable way.
+- Algorithm :ref:`ShowPossibleCells <algm-ShowPossibleCells>` now has a new output property, ``Cells``, that includes the cell information.
 - New algorithm :ref:`SaveMDHistoToVTK <algm-SaveMDHistoToVTK>` that saves a :ref:`MDHistoWorkspace <MDHistoWorkspace>` as a ``VTK`` file so that it can be visualized by ``Paraview``.
 - :ref:`IntegratePeaksShoeboxTOF <algm-IntegratePeaksShoeboxTOF>` has improved optimisation for shoebox position - would previously be centred on nearby stronger peaks if present.
-- Execution time of :ref:`IntegratePeaks1DProfile <algm-IntegratePeaks1DProfile>` has been sped up by an order of magnitude.
+- Execution time of :ref:`IntegratePeaks1DProfile <algm-IntegratePeaks1DProfile>` is at least 10 times faster.
```
Comment (thomashampson) [inline docs/source/release/v6.11.0/diffraction.rst:82]: It doesn't really make sense to say that the time of something is 10 times faster.
```suggestion
- Execution time of :ref:`IntegratePeaks1DProfile <algm-IntegratePeaks1DProfile>` has improved by a factor of ten.
```

File: docs/source/release/v6.11.0/mantidworkbench.rst (hunk 2 above)
Comment (thomashampson) [reply docs/source/release/v6.11.0/mantidworkbench.rst]: done

**Review by jhaigh0 — APPROVED** [review]

**Pull request merged**